### PR TITLE
Updates MODE to what I think they ought to be, if NeTEx is the master for the MODE Enum

### PR DESCRIPTION
**Pull request by ue71603**
Please take a good look, if this fits for you:
* Better descriptions (if you disagree, then we will have to change them)
* ACCESS MODE generalised also for TRANSFER
* Connection has a mode of its own now. Can we live with only one?

This is the summary of the discussion with  @Aurige  and should be discussed in the TRANSMODEL group as well.

@nick-knowles  Please also have a look

### CODE REVIEW

**Review by davidr1234 — APPROVED** [review]
Looks good to me.

**Review by Aurige — APPROVED** [review]

**Review by syversenkr — APPROVED** [review]
Agreeing to the proposed mode annotations and addition of TransferMode element

**Review by skinkie — APPROVED** [review]

File: xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd [hunk 1]
```diff
@@ -215,36 +219,98 @@ Rail transport, Roads and Road transport
...
+			</xsd:enumeration>
+			<xsd:enumeration value="lift">
+			  <xsd:annotation>
+			    <xsd:documentation>General Mode for Modes moved by cable. Especially lifts are not only vertical elevators. If a better specialisation applies (like cableway) use that.</xsd:documentation>
+			  </xsd:annotation>
+			</xsd:enumeration>
 			<xsd:enumeration value="selfDrive">
 				<xsd:annotation>
-					<xsd:documentation>See pti12_x.</xsd:documentation>
+					<xsd:documentation>For all alternative modes where a vehicle is driven by the user .See pti12_x.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd:301]:
```suggestion
					<xsd:documentation>For all alternative modes where a vehicle is driven by the user. See pti12_x.</xsd:documentation>
```

**Comment by Aurige** [comment]
Part 1 document Updated

**Pull request closed without merging**